pkgs/development/libraries/libpostal/default.nix

```
@@ -13,9 +44,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
 
-  configureFlags = [
-    "--disable-data-download"
-  ] ++ lib.optionals stdenv.hostPlatform.isAarch64 [ "--disable-sse2" ];
+  configureFlags = [ "--disable-data-download" ]
+    ++ lib.optionals stdenv.hostPlatform.isAarch64 [ "--disable-sse2" ]
+    ++ lib.optionals withData [ "--datadir=${data}/data" ];
```

I personally liked more the previous formatting. Could you maybe settle down on this:

```suggestion
  configureFlags = [
    "--disable-data-download"
  ] ++ lib.optionals stdenv.hostPlatform.isAarch64 [
    "--disable-sse2"
  ] ++ lib.optionals withData [
    "--datadir=${data}/data"
  ];
```

This is the formatting that best handles diffs in my experience.